src/dist/triple.rs

```diff
@@ -5,46 +5,138 @@ use regex::Regex;
 // the PartialTargetTriple.
 
 static LIST_ARCHS: &[&str] = &[
-    "i386",
-    "i586",
-    "i686",
-    "x86_64",
+    "aarch64",
```

Comment:
Just to check, where did you source these lists of current arches, OSes and envs?

Reply:
I copied all the targets from the [Platform Support section of the rustc book](https://doc.rust-lang.org/rustc/platform-support.html) and created the lists manually. I'm not sure if these lists already exist somewhere, so I created them manually.

Reply:
Sure, maybe add a docstring linking to a section in the book for each of these constants?